corehq/apps/hqadmin/management/commands/cchq_prbac_bootstrap.py

```diff
@@ -165,6 +165,7 @@ def ensure_roles(self, roles, dry_run=False):
         Role(slug=privileges.APP_USER_PROFILES,
              name='App User Profiles',
              description='Allows use of App User Profiles.'),
+        Role(slug=privileges.GEOCODER, name="Project's default location.", description=''),
```

Comment:
This and the other user-facing text should be something like "Address widget in Web Apps" although perhaps Kat or Dev will have a better suggestion on the ticket.

Reply:
Added name according to Kat
https://dimagi-dev.atlassian.net/browse/USH-212?focusedCommentId=93576